src/Components/DataEntry/RadioButton.vue

```diff
@@ -0,0 +1,324 @@
+		clip: rect(1px, 1px, 1px, 1px);
+		padding: 0;
+		border: 0;
+		height: 1px;
+		width: 1px;
+		overflow: hidden;
+
+		&:disabled {
+			cursor: not-allowed !important;
+			& + label {
+				cursor: not-allowed !important;
+
+				&:active {
+					pointer-events: none;
+				}
+				&.checked {
+					&:after {
+						background-color: $input-border-color !important;
+					}
+				}
+			}
+			& + label:before {
+				background-color: $gray100;
+			}
+		}
+
+		&:checked:not(:disabled) {
+			& + label:before {
+				background-color: $gray000;
+			}
+			& + label:after {
+				display: block;
+			}
+		}
+
+		& + label {
+			display: inline-flex;
+			flex-direction: row;
+			align-items: flex-start;
+			cursor: pointer;
+			position: relative;
+			> div {
+				display: inline-block;
+			}
+			&.checked {
+				&:after {
+					display: block;
+				}
+			}
+			&:before {
+				width: 20px;
+				height: 20px;
+				content: '';
+				background-color: $gray000;
+				background-position: center center;
+				border: 1px solid $input-border-color;
+				@include border-radius(20px);
+				display: inline-block;
+				margin-top: 3px;
+				margin-right: 8px;
+				box-sizing: border-box;
+			}
+			&:after {
+				content: '';
+				position: absolute;
+				top: 8px;
+				left: 5px;
+				width: 10px;
+				height: 10px;
+				@include border-radius(10px);
+				display: none;
+			}
+		}
+	}
+	&.primary {
+		input[type='radio'] {
+			&:hover:not(:disabled),
+			&:focus:not(:disabled) {
+				& + label:before {
+					background-color: $green000;
+					border-color: $green600;
+				}
+			}
+			& + label {
+				&.checked {
+					&:after {
+						background-color: $green600;
+					}
+				}
+			}
+			&:checked:not(:disabled) {
+				& + label:before {
+					border-color: $green600;
+				}
+				& + label:after {
+					background-color: $green600;
+				}
+			}
+		}
+	}
+	&.success {
+		input[type='radio'] {
+			&:hover:not(:disabled),
+			&:focus:not(:disabled) {
+				& + label:before {
+					background-color: $blue000;
+					border-color: $blue600;
+				}
+			}
+			& + label {
+				&.checked {
+					&:after {
+						background-color: $blue600;
+					}
+				}
+			}
+			&:checked:not(:disabled) {
+				& + label:before {
+					border-color: $blue600;
+				}
+				& + label:after {
+					background-color: $blue600;
+				}
+			}
+		}
+	}
+	&.secondary {
+		input[type='radio'] {
+			&:hover:not(:disabled),
+			&:focus:not(:disabled) {
+				& + label:before {
+					background-color: $gray000;
+					border-color: $gray600;
+				}
+			}
+			& + label {
+				&.checked {
+					&:after {
+						background-color: $gray600;
+					}
+				}
+			}
+			&:checked:not(:disabled) {
+				& + label:before {
+					border-color: $gray600;
+				}
+				& + label:after {
+					background-color: $gray600;
+				}
+			}
+		}
+	}
+	&.error {
+		input[type='radio'] {
+			&:hover:not(:disabled),
+			&:focus:not(:disabled) {
+				& + label:before {
+					background-color: $red000;
+					border-color: $red600;
```

Comment:
`$red600`이 반복되고 있는데, 변수 1개로 활용하면 유지보수하기에 더 좋을 것 같아요.
기존에 있는 `$error`를 사용하거나, 혹시 모를 커스텀에 대비해서 `$radio-error` 변수를 따로 만들어서 사용할 수도 있겠네요.
```suggestion
					border-color: $error;
```
```suggestion

                                      $radio-error: $red600;
					border-color: $radio-error;
```